### ISSUE TITLE
Minor readme suggestion: remove or modify `TF_LOG_PROVIDER=INFO`

### ISSUE THREAD

**Issue by dberenbaum** [iterative/terraform-provider-iterative#556]
Some people may find prepending `TF_LOG_PROVIDER=INFO` to every command foreign and confusing, and makes all the command examples look much longer and more complex. How important is this? If you feel it's needed in the readme, could you just have a note somewhere suggesting `export TF_LOG_PROVIDER=INFO`?

**Comment by 0x2b3bfa0** [comment]
Indeed, this is jut the tip of the iceberg for a bigger problem: pretending that we're using Terraform. 😅 

* https://github.com/iterative/terraform-provider-iterative/pull/423#pullrequestreview-901140497
* https://github.com/iterative/terraform-provider-iterative/pull/423#issuecomment-1060080669

(Marking as off–topic to avoid derailing the conversation)
